[2023-12-16T12:29:51.903+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_dag2.load_data_to_github manual__2023-12-16T07:29:18.835776+00:00 [queued]>
[2023-12-16T12:29:51.910+0500] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_dag2.load_data_to_github manual__2023-12-16T07:29:18.835776+00:00 [queued]>
[2023-12-16T12:29:51.910+0500] {taskinstance.py:1361} INFO - Starting attempt 1 of 2
[2023-12-16T12:29:53.235+0500] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): load_data_to_github> on 2023-12-16 07:29:18.835776+00:00
[2023-12-16T12:29:53.237+0500] {standard_task_runner.py:57} INFO - Started process 28819 to run task
[2023-12-16T12:29:53.240+0500] {standard_task_runner.py:84} INFO - Running: ['airflow', 'tasks', 'run', 'etl_dag2', 'load_data_to_github', 'manual__2023-12-16T07:29:18.835776+00:00', '--job-id', '1394', '--raw', '--subdir', 'DAGS_FOLDER/airflow_dag.py', '--cfg-path', '/tmp/tmp9ljvw4n8']
[2023-12-16T12:29:53.241+0500] {standard_task_runner.py:85} INFO - Job 1394: Subtask load_data_to_github
[2023-12-16T12:29:55.295+0500] {task_command.py:416} INFO - Running <TaskInstance: etl_dag2.load_data_to_github manual__2023-12-16T07:29:18.835776+00:00 [running]> on host amina-HP-ProBook-450-G2
[2023-12-16T12:29:55.702+0500] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='amina_bibi' AIRFLOW_CTX_DAG_ID='etl_dag2' AIRFLOW_CTX_TASK_ID='load_data_to_github' AIRFLOW_CTX_EXECUTION_DATE='2023-12-16T07:29:18.835776+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-12-16T07:29:18.835776+00:00'
[2023-12-16T12:30:00.854+0500] {python.py:194} INFO - Done. Returned value was: None
[2023-12-16T12:30:00.860+0500] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=etl_dag2, task_id=load_data_to_github, execution_date=20231216T072918, start_date=20231216T072951, end_date=20231216T073000
[2023-12-16T12:30:01.105+0500] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2023-12-16T12:30:01.121+0500] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
